src/content/3/en/part3b.md

```diff
@@ -101,23 +101,23 @@ If you decide to use [Fly.io](https://fly.io/) begin by installing their flyctl
 
 By default everyone gets two free virtual machines that can be used for running two apps at the same time.
 
-Note that the Fly.io instructions have only been added to this course on the 28th of August 2022. If you run into problems, please ask for help in Discord!
+Note that the Fly.io instructions have only been added to this course on the 28th of August 2022. If you run into problems, please ask for help on Discord!
 
 Start by [authenticating](https://fly.io/docs/hands-on/sign-in/) via command line with the command
 
 ```bash
 fly auth login
 ```
 
-*Note* if the command _fly_ does not work in your machine, you can try the longer version _flyctl_. Eg. on MacOS, both forms of the command work.
+*Note* if the command _fly_ does not work on your machine, you can try the longer version _flyctl_. Eg. on MacOS, both forms of the command work.
 
 Initializing an app happens by running the following command in the root directory of the app
 
 ```bash
 fly launch
 ```
 
-Give the app a name or let Fly.io autogenerate one. Pick a region where the app will be run. Do not create a postgres database for the app since it is not needed.
+Give the app a name or let Fly.io auto generate one. Pick a region where the app will be run. Do not create a postgress database for the app since it is not needed.
```

Comment:
I believe "postgres" refers to [PostgreSQL](https://www.postgresql.org/) so no error here?